app_config.py

```diff
@@ -1,21 +1,16 @@
 import os
 
-CLIENT_ID = "Enter_the_Application_Id_here" # Application (client) ID of app registration
+# Application (client) ID of app registration
+CLIENT_ID = os.getenv("CLIENT_ID")
+# Application's generated client secret: never check this into source control!
+CLIENT_SECRET = os.getenv("CLIENT_SECRET")
 
-CLIENT_SECRET = "Enter_the_Client_Secret_Here" # Placeholder - for use ONLY during testing.
-# In a production app, we recommend you use a more secure method of storing your secret,
-# like Azure Key Vault. Or, use an environment variable as described in Flask's documentation:
-# https://flask.palletsprojects.com/en/1.1.x/config/#configuring-from-environment-variables
-# CLIENT_SECRET = os.getenv("CLIENT_SECRET")
-# if not CLIENT_SECRET:
-#     raise ValueError("Need to define CLIENT_SECRET environment variable")
-
-AUTHORITY = "https://login.microsoftonline.com/common"  # For multi-tenant app
-# AUTHORITY = "https://login.microsoftonline.com/Enter_the_Tenant_Name_Here"
+# AUTHORITY = "https://login.microsoftonline.com/"  # For multi-tenant app
```

Comment:
Even if you currently choose to comment out this line, its demonstrative value should still contain the "common" otherwise the multi-tenant setup won't work.
```suggestion
# AUTHORITY = "https://login.microsoftonline.com/common"  # For multi-tenant app
```